Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspector.js

```diff
@@ -201,12 +202,26 @@ define([
         tileDebugLabelsPanelContents.appendChild(makeCheckbox('showRenderingStatistics', 'Rendering Statistics'));
         tileDebugLabelsPanelContents.appendChild(makeCheckbox('showMemoryUsage', 'Memory Usage (MB)'));
 
+        optimizationPanelContents.appendChild(makeCheckbox('skipLevelOfDetail', 'Skip Tile LODs'));
+        var optSSEContainer = document.createElement('div');
+        optSSEContainer.appendChild(makeRangeInput('skipScreenSpaceErrorFactor', 1, 50, 1, 'Skip Screen Space Error Factor'));
+        optimizationPanelContents.appendChild(optSSEContainer);
+        var optBaseScreenSpaceError = document.createElement('div');
+        optBaseScreenSpaceError.appendChild(makeRangeInput('baseScreenSpaceError', 0, 4096, 1, 'Screen space error before skipping LOD'));
+        optimizationPanelContents.appendChild(optBaseScreenSpaceError);
+        var optSkipLevelsContainer = document.createElement('div');
+        optSkipLevelsContainer.appendChild(makeRangeInput('skipLevels', 0, 10, 1, 'Min. levels to skip when loading tiles'));
+        optimizationPanelContents.appendChild(optSkipLevelsContainer);
+        optimizationPanelContents.appendChild(makeCheckbox('immediatelyLoadDesiredLOD', 'Load only tiles that meet the max. screen space error.'));
+        optimizationPanelContents.appendChild(makeCheckbox('loadSiblings', 'Load siblings of visible tiles.'));
```

Comment:
If possible, try to format so the slider and input box fit on the same line:

<img width="324" alt="formatting" src="https://cloud.githubusercontent.com/assets/915398/26787686/e86a8c38-49d8-11e7-86c4-917981d02924.PNG">